### ISSUE TITLE
assert dataset Assertion Error

### ISSUE THREAD

**Issue by sumorday** [huiyegit/T2I_CL#10]
![screen](https://user-images.githubusercontent.com/49804057/152763152-072b3092-c1f7-413f-8ba7-6dbc340c5b5a.png)

Hi. Is there something wrong with my data import? The path was changed in the cofig file, but it seemed that there was a problem reading the text file.

I saw the original attgan said that the data list should look like this：
data/birds 
Then my dataset is a list like this. Is it correct?

![list](https://user-images.githubusercontent.com/49804057/152763530-8d682d4d-3efd-44ff-9e5c-8c9641070dc9.png)

#if use cd command to explore the dataset:

!cd /content/T2I_CL/AttnGAN+CL/data/birds
!ls

attributes	    image_class_labels.txt  parts
bounding_boxes.txt  images		    README
classes.txt	    images.txt		    train_test_split.txt

 Is it correct?

**Comment by priyankaupadhyay090** [comment]
Code is unable to load train and test file names. you should keep train, test and text folder inside 'data/birds'. Problem should be solve after that

**Comment by priyankaupadhyay090** [comment]
> ![screen](https://user-images.githubusercontent.com/49804057/152763152-072b3092-c1f7-413f-8ba7-6dbc340c5b5a.png)
> 
> Hi. Is there something wrong with my data import? The path was changed in the cofig file, but it seemed that there was a problem reading the text file.
> 
> I saw the original attgan said that the data list should look like this： data/birds Then my dataset is a list like this. Is it correct?
> 
> ![list](https://user-images.githubusercontent.com/49804057/152763530-8d682d4d-3efd-44ff-9e5c-8c9641070dc9.png)
> 
> #if use cd command to explore the dataset:
> 
> !cd /content/T2I_CL/AttnGAN+CL/data/birds !ls
> 
> attributes image_class_labels.txt parts bounding_boxes.txt images README classes.txt images.txt train_test_split.txt
> 
> Is it correct?

Hey I got the same error and was able to solve it.

so basically Code is unable to load train and test filenames.pickle. you should keep train, test and text folder inside 'data/birds'. Problem should be solve after that

**Comment by sumorday** [comment]
> 
![屏幕快照 2022-02-08 下午3 04 55](https://user-images.githubusercontent.com/49804057/152927936-223de044-e89c-4eb5-a2a0-89589a367053.png)

thanks for the reply. I can put them all under the birds folder. Among them, bounding_boxes and images.txt, colab prompted me to put them in the folder CUB_200_2011.

I first downloaded the compressed package of CUB_200_2011. and decompressed it. I tried putting all txt, train files into birds folder but it doesn't work. May I look at that list of your datasets? Thank you soooo much:)
